variables.tf

```diff
@@ -91,6 +102,7 @@ variable "tfe_workspace" {
     file_triggers_enabled          = optional(bool, true)
     global_remote_state            = optional(bool, false)
     name                           = optional(string, null)
+    permissions_boundary_arn       = optional(string, null)
```

Comment:
this is old?

Reply:
This is new, as we are passing in the arn from avm to workspace. Just a variable defined.